examples/loopback.rs

```diff
@@ -57,12 +60,12 @@ mod mock {
             Clock(Arc::new(AtomicUsize::new(0)))
         }
 
-        pub fn advance(&self, millis: u64) {
-            self.0.fetch_add(millis as usize, Ordering::SeqCst);
+        pub fn advance(&self, duration: Duration) {
+            self.0.fetch_add(duration.total_millis() as usize, Ordering::SeqCst);
         }
 
-        pub fn elapsed(&self) -> u64 {
-            self.0.load(Ordering::SeqCst) as u64
+        pub fn elapsed(&self) -> Instant {
+			Instant::from_millis(self.0.load(Ordering::SeqCst) as i64)
```

Comment:
something weird happened with indentation

Reply:
That is strange... somehow I inserted a tab here. Not sure how that happened.